recipe/meta.yaml

```diff
@@ -17,9 +17,9 @@ requirements:
   build:
     - python
     - pip
-    - setuptools>=38.6.0 
-    - wheel>=0.31.0 
-    - twine>=1.11.0
+    - setuptools >=38.6.0 
+    - wheel >=0.31.0 
+    - twine >=1.11.0
```

Comment:
Thanks. My bad :/

Reply:
Not your bad. Conda should be able to take `pip` standards...
(Although I haven't tested if the space is still needed.)